kubeflow/relations.tf

```diff
@@ -0,0 +1,490 @@
+
+resource "juju_integration" "argo_controller_minio" {
+  model = var.create_model ? juju_model.kubeflow[0].name : local.model_name
+
+  application {
+    name     = module.argo_controller.app_name
+    endpoint = "object-storage"
```

Comment:
In general, the endpoint should be provided by the module:
```suggestion
    endpoint = module.argo_controller.requires.object-storage
```

This of course requires an update to the module.

Reply:
The modules already support this so let me update it.

Reply:
updated